.github/workflows/daily-qa.yml

```diff
@@ -27,23 +27,27 @@ jobs:
     runs-on: ubuntu-latest
     outputs:
       latest-version: ${{ env.LATEST_VERSION }}
-      previous-latest-version: ${{ env.PREVIOUS_LATEST_VERSION }}
-      previous-previous-latest-version: ${{ env.PREVIOUS_PREVIOUS_LATEST_VERSION }}
+      second-last-version: ${{ env.SECOND_LAST_VERSION }}
+      third-last-version: ${{ env.THIRD_LAST_VERSION }}
+      fourth-last-version: ${{ env.FOURTH_LAST_VERSION }}
     steps:
       - uses: actions/checkout@v4
       - run: git fetch --tags
       - run: |
-          latest=$(git tag | grep -E '^[0-9]+\.[0-9]+\.[0-9]+$' | sed -E -e 's/\.[0-9]+$//' | sort -Vr | head -n1)
-          previous=$(git tag | grep -E '^[0-9]+\.[0-9]+\.[0-9]+$' | sed -E -e 's/\.[0-9]+$//' | grep -v "${latest}" | sort -Vr | head -n1)
-          previous_previous=$(git tag | grep -E '^[0-9]+\.[0-9]+\.[0-9]+$' | sed -E -e 's/\.[0-9]+$//' | grep -v "${latest}" | grep -v "${previous}" | sort -Vr | head -n1)
-          if [ ! -z $latest ] && [ ! -z $previous ] && [ ! -z $previous_previous ]; then
-            echo "Successfully computed latest versions: ${latest} and ${previous} and ${previous_previous}"
+          versions=$(git tag | grep -E '^[0-9]+\.[0-9]+\.[0-9]+$' | sed -E -e 's/\.[0-9]+$//' | sort -Vr | uniq)
+          latest=$(echo "$versions" | head -n1)
+          second_last=$(echo "$versions" | head -n2)
```

Comment:
❓ @Zelldon AFAIK, `head -nX` should retrieve first `X` lines of the result instead of the `X`th line. I didn't test it but might result in different output. Maybe I am missing something here. Is it possible to test the change?

Reply:
Yep you're right. I will fix it. https://github.com/camunda/zeebe/pull/16125#discussion_r1469340080